### PR TITLE
[MSG] Fix for silly bug in ArtworkPreview #trivial

### PR DESCRIPTION
**Pull request by sarahscott**
A cautionary tale.

This is a fix for artworks without dates - `this.props.artwork.date` was an empty string in this case instead of a `null`, resulting in this bug:

simulator screen shot jul 12 2017 5 02 22 pm

### CODE REVIEW

File: Example/Emission.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -558,7 +558,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
+			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
```
Comment (sarahscott) [inline Example/Emission.xcodeproj/project.pbxproj]: Not sure what this is about.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Yup, this has happened before :/

LGTM, seeing as @mzikherman is on vacay, I’ll take this.

**Comment by alloy** [comment]
This could do with a CHANGELOG entry, though.

**Comment by alloy** [comment]
👍

**Pull request merged**